packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -36,15 +28,19 @@ export const createSignatureHeaders = async ({
   if (request.body) {
     components.push('content-digest', 'content-length', 'content-type')
   }
-  const { headers } = await httpsig.sign(request, {
-    components,
-    parameters: {
-      created: Math.floor(Date.now() / 1000)
```

Comment:
No need to specify `created`'s value. The library is going to automatically generate if it's passed as a parameter.